quadrat/index.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * The main template file
+ *
+ * This is the most generic template file in a WordPress theme
+ * and one of the two required files for a theme (the other being style.css).
+ * It is used to display a page when nothing more specific matches a query.
+ * E.g., it puts together the home page when no home.php file exists.
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-hierarchy/
+ *
+ * @package Quadrat
+ * @since 1.0.0
+ */
+	// the header
+	echo do_blocks( file_get_contents( 'templates/header.html' ) );
+
+	// the query
+	echo do_blocks( file_get_contents( 'templates/query.html' ) );
+
+	// the footer
+	echo do_blocks( file_get_contents( 'templates/footer.html' ) );
```

Comment:
This is very smart! it would solve our issues with the FSE being available to users, but so far it doesn't seem to be working on my local install. It seems like the theme is using the index.html template from blank canvas instead of this one. Does that happen to you too?

Reply:
I went ahead and made a PR to fix that: https://github.com/Automattic/themes/pull/3651

Reply:
Thank you for making that fix and opening an issue in GB about the child theme bug!